test/Model/WipeMessageTest.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * WipeMessageTest
+ *
+ * PHP version 7.4
+ *
+ * @category Class
+ * @package  IONOS\MailNotificationAPI\Client
+ * @author   OpenAPI Generator team
+ * @link     https://openapi-generator.tech
+ */
+
+/**
+ * Event Notification Handler
+ *
+ * This is the API client for the Mail Notification API
+ *
+ * The version of the OpenAPI document: 1.0.0-20241129122727
+ * Generated by: https://openapi-generator.tech
+ * Generator version: 7.10.0
+ */
+
+/**
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Please update the test case below to test the model.
+ */
+
+namespace IONOS\MailNotificationAPI\Client\Test\Model;
+
+use PHPUnit\Framework\TestCase;
+
+/**
+ * WipeMessageTest Class Doc Comment
+ *
+ * @category    Class
+ * @description WipeMessage
+ * @package     IONOS\MailNotificationAPI\Client
+ * @author      OpenAPI Generator team
+ * @link        https://openapi-generator.tech
+ */
+class WipeMessageTest extends TestCase
+{
+
+    /**
+     * Setup before running any test case
+     */
+    public static function setUpBeforeClass(): void
+    {
+    }
+
+    /**
+     * Setup before running each test case
+     */
+    public function setUp(): void
+    {
+    }
+
+    /**
+     * Clean up after running each test case
+     */
+    public function tearDown(): void
+    {
+    }
+
+    /**
+     * Clean up after running all test cases
+     */
+    public static function tearDownAfterClass(): void
+    {
+    }
+
+    /**
+     * Test "WipeMessage"
+     */
+    public function testWipeMessage()
+    {
+        // TODO: implement
+        self::markTestIncomplete('Not implemented');
+    }
+
+    /**
+     * Test attribute "deviceName"
+     */
+    public function testPropertyDeviceName()
+    {
+        // TODO: implement
+        self::markTestIncomplete('Not implemented');
+    }
+
+    /**
+     * Test attribute "language"
+     */
+    public function testPropertyLanguage()
+    {
+        // TODO: implement
+        self::markTestIncomplete('Not implemented');
+    }
+
+    /**
+     * Test attribute "senderUserId"
+     */
+    public function testPropertySenderUserId()
+    {
+        // TODO: implement
+        self::markTestIncomplete('Not implemented');
```

Comment:
Let's drop all unit test infrastructure as long as the **generated tests are no-op**?

Reply:
(...and tests too?)

Reply:
dropped test generation